### ISSUE TITLE
missing dependency symfony/options-resolver

### ISSUE THREAD

**Issue by 2binfree** [FriendsOfSymfony/FOSRestBundle#1909]
Using configuration
```
body_converter:
    enabled: true
    validate: true
    validation_errors_argument: violations
```
return a missing class `OptionsResolver` into class `RequestBodyParamConverter`.
I work with an application build with Symfony 4.1 and symfony skeleton for minimal component integration.
After fos_rest install, i had to need to manual add `symfony/options-resolver`.
Is where missing requirement into composer.json ?

**Comment by xabbuh** [comment]
Since the `RequestBodyParamConverter` is the only class depending on the OptionsResolver component I think the best solution is to throw a meaningful exception when the option is enabled without the component being installed. Would you like to create a PR?

**Comment by 2binfree** [comment]
yes, i can do that

**Comment by xabbuh** [comment]
closing here as this was fixed by #1912
